ui/imports/shared/views/DeviceSyncingView.qml

```diff
@@ -180,6 +183,17 @@ Item {
                 spacing: 4
                 clip: true
 
+//                model: SortFilterProxyModel {
```

Comment:
is it intended to be removed?

Reply:
Also shouldn't be here, dropped this commit.